nusacrowd/nusa_datasets/indonesian_wsd/indonesian_wsd.py

```diff
@@ -0,0 +1,201 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+from nusacrowd.utils.constants import Tasks
+from nusacrowd.utils import schemas
+
+import datasets
+import json
+
+from nusacrowd.utils.configs import NusantaraConfig
+
+_CITATION = """\
+@inproceedings{mahendra-etal-2018-cross,
+    title = "Cross-Lingual and Supervised Learning Approach for {I}ndonesian Word Sense Disambiguation Task",
+    author = "Mahendra, Rahmad  and
+      Septiantri, Heninggar  and
+      Wibowo, Haryo Akbarianto  and
+      Manurung, Ruli  and
+      Adriani, Mirna",
+    booktitle = "Proceedings of the 9th Global Wordnet Conference",
+    month = jan,
+    year = "2018",
+    address = "Nanyang Technological University (NTU), Singapore",
+    publisher = "Global Wordnet Association",
+    url = "https://aclanthology.org/2018.gwc-1.28",
+    pages = "245--250",
+    abstract = "Ambiguity is a problem we frequently face in Natural Language Processing. Word Sense Disambiguation (WSD) is a task to determine the correct sense of an ambiguous word. However, research in WSD for Indonesian is still rare to find. The availability of English-Indonesian parallel corpora and WordNet for both languages can be used as training data for WSD by applying Cross-Lingual WSD method. This training data is used as an input to build a model using supervised machine learning algorithms. Our research also examines the use of Word Embedding features to build the WSD model.",
+}
+"""
+
+_LANGUAGES = ["ind"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LOCAL = False
+
+_DATASETNAME = "indonesian_wsd"
+
+_DESCRIPTION = """\
+Word Sense Disambiguation (WSD) is a task to determine the correct sense of an ambiguous word.
+The training data was collected from news websites and manually annotated. The words in training data were processed using the morphological analysis to obtain lemma.
+The features being used were some words around the target word (including the words before and after the target word), the nearest verb from the
+target word, the transitive verb around the target word, and the document context. 
+"""
+
+_HOMEPAGE = ""
+
+_LICENSE = "Unknown"
+
+_URLS = {
+    _DATASETNAME: "https://github.com/rmahendra/Indonesian-WSD/raw/master/dataset-clwsd-ina.zip",
```

Comment:
In here, I saw there's a `sense-ID` that is also stated [in the paper](https://aclanthology.org/2018.gwc-1.28.pdf), but it's not loaded in the implemented dataloader. Could you confirm whether this is needed in the WSD task?

Reply:
`sense-ID` is like `00125993` yap based on the data, isn't it? @bryanwilie 

<img width="680" alt="Screen Shot 2022-09-24 at 21 19 23" src="https://user-images.githubusercontent.com/11547086/192103079-0bd951e9-f716-4631-922e-c68f58129bb5.png">

I think for this can be implemented

Reply:
For `nusantara` source I still have no any idea about the fit schema that can be applied with data `text`, `label`, and `sense_id`. Any suggestions maybe?

Reply:
After I read the paper I think `sense_id` is an important thing

Reply:
Yes yes I think that's the sense-id. I'm also new to this, but when I read the paper, I get that this is important 👍🏻.

Reply:
okay just add an updates based on the discussion in slack, kindly check again @bryanwilie, thank you!

Reply:
Checked and commented below. Thank you @muhsatrio!